Assets/Scripts/Gamestate/PlayerManager.cs

```diff
@@ -149,7 +156,8 @@ private void UpdateAimTarget(GunStats stats)
     {
         Vector3 cameraCenter = inputManager.transform.position;
         Vector3 cameraDirection = inputManager.transform.forward;
-        if (Physics.Raycast(cameraCenter, cameraDirection, out RaycastHit hit, maxHitDistance, hitMask))
+        Vector3 startPoint = cameraCenter + cameraDirection * targetStartOffset;
```

Comment:
Does this create issues when players are facing off real close?

Reply:
Yes, but only when the gun is so close that its "inside" the other player, at which point I think it's fine anyway 😅